docs/src/content/docs/fr/components/link-buttons.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Boutons de liaison
+description: Apprenez à créer des boutons de liaison dans Starlight pour des liens visuellement distincts qui appellent à l'action.
+---
+
+import { LinkButton } from '@astrojs/starlight/components';
+
+Pour afficher des liens visuellement distincts qui appellent à l'action (également connus sous le nom de « call-to-action » ou « CTA » en anglais), utilisez le composant `<LinkButton>`.
```

Comment:
From the [previous](https://github.com/withastro/starlight/blob/8112d8a29ead9c56640a6918c7bfaa5b26c74b17/docs/src/content/docs/fr/guides/components.mdx?plain=1#L235-L265) names and synonyms.